src/App.js

```diff
@@ -1,11 +1,32 @@
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] OPTIONAL
You have **disabled linter check** on your code. Please enable eslint linter check on your code to ensure that your code syntax is well structured and robust.